tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -511,8 +505,14 @@ final class TestingServer(
     require(server.bspSession.isDefined, "Build server did not initialize")
   }
 
-  def toPath(filename: String): AbsolutePath =
+  def toPath(filename: String): AbsolutePath = {
     TestingServer.toPath(workspace, filename, virtualDocSources)
+  }
+
+  def toPathFromSymbol(symbol: String, filename: String): AbsolutePath = {
+    workspaceSymbol(symbol)
```

Comment:
We need to first find the particular symbol in order to cache the location of the jar. Could try and simplify it further, but I don't think it's neccessary currently.